src/handlers/choosingArticle.js

```diff
@@ -191,28 +192,90 @@ export default async function choosingArticle(params) {
 
       // Track not yet reply Articles.
       ga(userId, { ec: 'Article', ea: 'NoReply', el: selectedArticleId });
-
       const text =
         '【跟編輯說您的疑惑】\n' +
         '抱歉這篇訊息還沒有人回應過唷！\n' +
         '\n' +
         '若您覺得這是一則謠言，請指出您有疑惑之處，說服編輯這是一份應該被闢謠的訊息。\n' +
         '\n' +
-        '請按左下角「⌨️」鈕，把「為何您會覺得這是一則謠言」的理由傳給我們，幫助闢謠編輯釐清您有疑惑之處；\n' +
-        '若想跳過，請按「我不想填理由」按鈕。';
+        '請按左下角「⌨️」鈕，把「為何您會覺得這是一則謠言」的理由傳給我們，幫助闢謠編輯釐清您的疑惑；\n' +
+        '若想跳過，請輸入「n」。';
+      let accountName = process.env.LINE_AT_NAME || 'cofacts';
```

Comment:
1. Let's add `LINE_AT_NAME` to README so that people gets to know we should setup this. Specify that it's the LINE ID without `@`.
2. How about using `const` here?